mon-pix/tests/unit/services/oidc-identity-providers-test.js

```diff
@@ -284,4 +284,39 @@ module('Unit | Service | oidc-identity-providers', function (hooks) {
       });
     });
   });
+
+  module('shouldDisplayAccountRecoveryBanner', function () {
+    test('return true if sso is in list', async function (assert) {
+      // given
+      class FeatureTogglesStub extends Service {
+        featureToggles = { isNewAccountRecoveryEnabled: true };
+      }
+      this.owner.register('service:featureToggles', FeatureTogglesStub);
+      const oidcIdentityProvidersService = this.owner.lookup('service:oidcIdentityProviders');
+
+      // when
+      const shouldDisplayAccountRecoveryBanner =
+        await oidcIdentityProvidersService.shouldDisplayAccountRecoveryBanner('FER');
+
+      // then
+      assert.true(shouldDisplayAccountRecoveryBanner);
+    });
+    context('when feature toggle is set to false', function () {
```

Comment:
`context` n’existe pas dans qunit, à la place il faut utiliser `module` :
```suggestion
    module('when feature toggle is set to false', function () {
```